phpBB/phpbb/controller/helper.php

```diff
@@ -184,6 +184,8 @@ public function route($route, array $params = array(), $is_amp = true, $session_
 	* @param string $message The error message
 	* @param int $code The error code (e.g. 404, 500, 503, etc.)
 	* @return Response A Response instance
+	*
+	* @deprecated 3.1.3 (To be removed: 3.3.0) Use exceptions instead.
```

Comment:
Don't think that belongs in the depreciation notice, we should document exceptions in general though and when to use which.